Tests/Merge-GitCommit.Tests.ps1

```diff
@@ -309,7 +309,7 @@ Describe 'Merge-GitCommit.when there are conflicts' {
     GivenFile 'conflict' 'master'
     WhenMerging 'develop' 
     ThenMergeStatus -Is ([LibGit2Sharp.MergeStatus]::Conflicts)
-    ThenFileContentIs 'conflict' ("<<<<<<< HEAD`nmaster`n=======`ndevelop`n>>>>>>> {0}`n" -f (Get-GitCommit -RepoRoot (Get-RepoRoot) -Revision 'develop').Sha)
+    ThenFileContentIs 'conflict' ("<<<<<<< HEAD`r`nmaster`r`n=======`r`ndevelop`r`n>>>>>>> {0}`r`n" -f (Get-GitCommit -RepoRoot (Get-RepoRoot) -Revision 'develop').Sha)

```

Comment:
This is a fragile test that is bound to break when running on different platforms, for example you had to fix it to work for you but now it'll break on the build server. Let's simplify this test. We don't really care about the exact content of the file, we merely need to check for the expected commit hash to indicate we merged in the expected direction.

Can you add a new test function `ThenFileContentMatch` that looks something like this:
```
function ThenFileContentMatch
{
    param(
        $Name,
        $ExpectedRegex
    )

    It 'should have file with correct content' {
        $path = Join-Path -Path (Get-RepoRoot) -ChildPath $Name
        Get-Content -Raw -Path $path | Should -Match $ExpectedRegex
    }
}
```

And then change this line to the following:
```
ThenFileContentMatch 'conflict' (Get-GitCommit -RepoRoot (Get-RepoRoot) -Revision 'develop').Sha
```

Reply:
Agreed - good idea.

Reply:
Done, works.